docs/products/opensearch/get-started.md

```diff
@@ -19,20 +22,39 @@ supports integrations for logs and monitoring.
 
 Ensure you have the following before getting started:
 
+<Tabs groupId="group1">
```

Comment:
If I select the "Console" tab here for prerequisites, perhaps I'd like the same (the "Console" tab) previewed automatically in the "Connect to your service" section.